src/pages/overlay/hooks/useSettings.tsx

```diff
@@ -38,6 +38,12 @@ const settings = {
     },
     configurable: false,
   },
+  disableCardEffects: {
+    title: "Disable Ambassador Card Effects",
+    type: "boolean",
+    process: (value: any) => !!value,
+    configurable: true,
+  },
```

Comment:
Nit, this should probably go above `disableOverlayHiding`, so that the regular user-configurable settings are grouped together?